packages/kbn-pm/src/utils/bazel/setup_remote_cache.ts

```diff
@@ -64,56 +43,17 @@ export async function setupRemoteCache(repoRootPath: string) {
 
   const settingsPath = resolve(repoRootPath, '.bazelrc.cache');
 
-  // Checks if we should upgrade the servers used on .bazelrc.cache
-  //
-  // NOTE: this can be removed in the future once everyone is migrated into the new servers
-  if (await migrateToNewServersIfNeeded(settingsPath)) {
-    return;
-  }
-
-  if (existsSync(settingsPath)) {
-    log.debug(`[bazel_tools] remote cache settings already exist, skipping`);
-    return;
-  }
-
-  if (!(await isVaultAvailable())) {
-    log.info('[bazel_tools] vault is not available, unable to setup remote cache settings.');
-    log.info('[bazel_tools] building packages will work, but will be slower in many cases.');
-    log.info('[bazel_tools] use the following guide or reach out to Operations for assistance');
-    log.info('[bazel_tools] https://github.com/elastic/infra/tree/master/docs/vault');
-    return;
-  }
-
-  let apiKey = '';
-
-  try {
-    const { stdout } = await spawn(
-      'vault',
-      ['read', '-field=readonly-key', 'secret/ui-team/kibana-bazel-remote-cache'],
-      {
-        stdio: 'pipe',
-      }
-    );
-    apiKey = stdout.trim();
-  } catch (ex: unknown) {
-    log.info(
-      '[bazel_tools] unable to read bazel remote cache key from vault, are you authenticated?'
-    );
-    log.info('[bazel_tools] building packages will work, but will be slower in many cases.');
-    log.info('[bazel_tools] reach out to Operations if you need assistance with this.');
-    log.info(`[bazel_tools] ${ex}`);
-
+  // Checks if we should upgrade or install the config file
+  if (await upToDate(settingsPath)) {
+    log.debug(`[bazel_tools] remote cache config already exists and is up-to-date, skipping`);
     return;
   }
 
   const contents = dedent`
-    # V1 - This file is automatically generated by 'yarn kbn bootstrap'
+    # V2 - This file is automatically generated by 'yarn kbn bootstrap'
     # To regenerate this file, delete it and run 'yarn kbn bootstrap' again.
-    build --bes_results_url=https://app.buildbuddy.io/invocation/
-    build --bes_backend=grpcs://remote.buildbuddy.io
-    build --remote_cache=grpcs://remote.buildbuddy.io
-    build --remote_timeout=3600
-    build --remote_header=${apiKey}
+    build --remote_cache=https://storage.googleapis.com/kibana-local-bazel-remote-cache
+    build --noremote_upload_local_results
```

Comment:
`build --incompatible_remote_results_ignore_disk` should be used together with `build --noremote_upload_local_results` to be able to have a combined cache (local + remote).